### ISSUE TITLE
Error reading role: pq: permission denied for relation pg_shadow

### ISSUE THREAD

**Issue by milosgajdos83** [terraform-providers/terraform-provider-postgresql#67]
### Terraform Version
```
Terraform v0.11.11
+ provider.aws v1.60.0
+ provider.postgresql v0.2.0
```

### Affected Resource(s)
```
postgresql_role
```

### Terraform Configuration Files
```hcl
provider "postgresql" {
  host            = "${var.host}"
  port            = "${var.port}"
  username        = "${var.admin_username}"
  password        = "${var.admin_password}"
  sslmode         = "require"
  connect_timeout = 15
}

resource "postgresql_role" "db" {
  name     = "${var.role_name}"
  password = "${var.role_password}"
  login    = true
}

resource "postgresql_database" "db" {
  name  = "${var.db_name}"
  owner = "${var.db_owner}"
}
```

### Debug Output
```
Error: Error refreshing state: 1 error(s) occurred:

2019/02/26 23:05:33 [DEBUG] plugin: waiting for all plugin processes to complete...
* module.db.module.postgres.postgresql_role.db: 1 error(s) occurred:

* module.db.module.postgres.postgresql_role.db: postgresql_role.db: Error reading role: pq: permission denied for relation pg_shadow
```

### Expected Behavior
`postgresql` role gets created without any errors.

### Actual Behavior
The `postgresql` database does get created and so does the `postgresql` role as I can use it to log on to the database, but it seems like RDS `master_user` tries to read `pg_shadow` table and it does not have permissions to do so which is super weird

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
This is AWS RDS cluster:
* `engine` :  aurora-postgresql
* `enginve_version`: 9.6.8

**Comment by mlehner616** [comment]
I can confirm this behavior and that 0.2 does not appear to be useable right now with RDS. In RDS, we don't have access to `pg_shadow` if I remember correctly.

I began receiving this same error after my version pinning was left a little too liberal with `~> 0.1.` Pinning back to `~> 0.1.3, < 0.2` fixed the error for now.

**Comment by cyrilgdn** [comment]
Hi @milosgajdos83 ,

Thanks for opening this issue, I allow myself to close it as it's a duplicate of #64 .

I already  created a PR to fix this problem: #66 Feel free to review it if you want.
A 0.2.1 will be released with this fix in the next two days.

**Comment by cyrilgdn** [comment]
Hi @milosgajdos83 @mlehner616 ,

[Version 0.2.1](https://github.com/terraform-providers/terraform-provider-postgresql/releases/tag/v0.2.1),  has been released today .

You can add `superuser = false` in your provider configuration so it will not try to refresh password from your Postgres server.

example:
```
provider "postgresql" {
  host = "test-provider.xxxxxxxxxxx.eu-central-1.rds.amazonaws.com"
  port = "5432"

  username  = "postgres"
  password  = "xxx"
  superuser = false
}
```

See #66
